Automattic/wp-calypso
# Show different support articles on My Home according to site intent

#### Changes proposed in this Pull Request

* Use site_intent to conditionally show specific Support articles on My Home, tailored to bloggers' needs.

| Before | After |
| - | - |
| ![image](https://user-images.githubusercontent.com/13596067/143409664-8a6dc64d-f0f5-42c1-ae8a-21ccec854bc8.png) | ![image](https://user-images.githubusercontent.com/13596067/143409620-2cf28fa6-f8bc-407c-be7b-36edb917e694.png) |

#### Testing instructions



* Apply D70637-code to your sandbox and ensure your API is pointed to your sandbox
* Go to `/home/<your_site>` where the site should be "write" intent
* Check the support articles is the same as https://github.com/Automattic/wp-calypso/issues/58302#issuecomment-976759601 mentioned
* Repeat the first step and use the site with "build" intent or the site without site intent
* Check the support articles keep the same



Related to https://github.com/Automattic/wp-calypso/issues/58302
